src/website/app/demos/shared/renderPropDescription.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-irregular-whitespace */
+export default `The callback function will be invoked with an object parameter
+containing the following information and utilities to help customize your
+implementation.
+
+* Props to be spread to your custom component
+    * Note that these may not all be necessary for your implementation, or even
+    valid DOM attributes, so you’ll want to apply them judiciously.  _If your
```

Comment:
Is it okay that these are not all valid DOM attributes?  It sure would be nice if they were.  What would it take to make it so?

Reply:
I think it is probably okay since Glamorous will filter them.  In most cases, user can just spread them and not worry about it.